File: Python Word Examples/EncryptApiDemo/encryptDocument.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.encrypt_api.EncryptApi(configuration)

file_name = 'EncryptDocument.docx'
folder = 'input'
new_password='123'
dest_file_path = 'output/EncryptDocument.docx'
api.encrypt_document(file_name, new_password=new_password, folder=folder, dest_file_path=dest_file_path)

File: PHP Word Examples/EncryptApiDemo/encryptDocument.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\EncryptApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new EncryptApi($configuration);

$fileName = "EncryptDocument.docx";
$folder = "input";
$storage = null;
$old_password = null;
$new_password = '123';
$dest_file_path = "output/EncryptDocument.docx";
$apiInstance->encryptDocument($fileName, $dest_file_path, $folder, $storage, $old_password, $new_password);